### PR TITLE
TransactionService and TransactionController en InvoiceGenerator added

### PR DESCRIPTION
**Pull request by ali-saad-dev**
TransactionService and TransactionController en InvoiceGenerator added

### CODE REVIEW

**Comment by ali-saad-dev** [comment]
[Add option to manage transactions](https://trello.com/c/kUj6r40n/19-add-option-to-manage-transactions)

File: src/main/java/nl/novi/serviceconnect/controllers/TransactionController.java [hunk 1]
```diff
@@ -38,31 +40,31 @@ public ResponseEntity<List<TransactionOutputDto>> getAllTransaction() {
...
-//    public ResponseEntity<ServiceInputDto> deleteService(@PathVariable Long id) {
-//        try {
-//            service.deleteService(id);
-//            return ResponseEntity.ok().build();
-//        } catch (RecordNotFoundException e) {
-//            return ResponseEntity.notFound().build();
-//        }
-//    }
+    @GetMapping("/{id}")
+    public ResponseEntity<TransactionOutputDto> getTransactionId(@PathVariable Long id) {
```
Comment (nairu777) [inline src/main/java/nl/novi/serviceconnect/controllers/TransactionController.java:44]: getTransactionId = getTransactionById

File: src/main/java/nl/novi/serviceconnect/controllers/TransactionController.java [hunk 2]
```diff
@@ -38,31 +40,31 @@ public ResponseEntity<List<TransactionOutputDto>> getAllTransaction() {
...
+        TransactionOutputDto transactionId = TransactionService.getTransactionById(id);
```
Comment (nairu777) [inline src/main/java/nl/novi/serviceconnect/controllers/TransactionController.java:45]: transactionId = transaction

Comment (ali-saad-dev) [reply src/main/java/nl/novi/serviceconnect/controllers/TransactionController.java:45]: done

**Pull request merged**